std/string.d

```diff
@@ -1966,10 +1972,16 @@ auto representation(Char)(Char[] s) @safe pure nothrow @nogc
 }
 
 
-/++
-    Capitalize the first character of $(D s) and convert the rest of $(D s)
-    to lowercase.
- +/
+/**
+ * Capitalize the first character of $(D s) and convert the rest of $(D s)
```

Comment:
You're write, it does. I was on mobile and couldn't view the whole file. Never mind me.